install_config/persistent_storage/persistent_storage_glusterfs.adoc

```diff
@@ -129,18 +129,185 @@ It is recommended to deploy both groups to avoid potential impacts on
+
+
+openshift_master_cluster_hostname=node101.redhat.com
+openshift_master_cluster_public_hostname=node101.redhat.com
+
+[masters]
+node101.redhat.com
+
+[etcd]
+node101.redhat.com
+
+[nodes]
+node101.redhat.com openshift_node_group_name="node-config-master"
+node102.redhat.com openshift_node_group_name="node-config-infra"
+node103.redhat.com openshift_node_group_name="node-config-compute"
+node104.redhat.com openshift_node_group_name="node-config-compute"
+node105.redhat.com openshift_node_group_name="node-config-compute"
+node106.redhat.com openshift_node_group_name="node-config-compute"
+node107.redhat.com openshift_node_group_name="node-config-compute"
+node108.redhat.com openshift_node_group_name="node-config-compute"
+
+[glusterfs]
+node103.redhat.com glusterfs_zone=1 glusterfs_devices='["/dev/sdd"]'
+node104.redhat.com glusterfs_zone=2 glusterfs_devices='["/dev/sdd"]'
+node105.redhat.com glusterfs_zone=3 glusterfs_devices='["/dev/sdd"]'
+
+[glusterfs_registry]
+node106.redhat.com glusterfs_zone=1 glusterfs_devices='["/dev/sdd"]'
+node107.redhat.com glusterfs_zone=2 glusterfs_devices='["/dev/sdd"]'
+node108.redhat.com glusterfs_zone=3 glusterfs_devices='["/dev/sdd"]'
+
+----
+
+[[install-advanced-installer-host-variables]]
+==== Host variables
+
+Each host in the `glusterfs` and `glusterfs_registry` groups must have the
+`glusterfs_devices` variable defined. This variable defines the list of block
+devices that will be managed as part of the GlusterFS cluster. You must have at
+least one device, which must be bare, with no partitions or LVM PVs.
+
+You can also define the following variables for each host. If they are defined,
+these variables further control the host configuration as a GlusterFS node:
+
+- `glusterfs_cluster`: The ID of the cluster this node belongs to.
+- `glusterfs_hostname`: A host name or IP address to be used for internal
+  GlusterFS communication.
+- `glusterfs_ip`: The IP address that the pods use to communicate with
+the GlusterFS node.
+- `glusterfs_zone`: A zone number for the node. Within the cluster, zones
+determine how to distribute the bricks of GlusterFS volumes.
+
+[[install-advanced-installer-role-variables]]
+==== Role variables
+
+To control the integration of a GlusterFS cluster into a new or existing
+{product-title} cluster, you can also define a number of role variables,
+which are stored in the inventory file. Each role variable also has a
+corresponding variable to optionally configure a separate GlusterFS cluster
+for use as storage for an integrated Docker registry.
+
+[[install-advanced-installer-image-name]]
+==== Image name and version tag variables
+
+To prevent {product-title} pods from upgrading after an outage leading to a cluster
+with different {product-title} versions, it is recommended that you specify the
+image name and version tags for all containerized components. These variables are:
 
 - `openshift_storage_glusterfs_image`
 - `openshift_storage_glusterfs_block_image`
```

Comment:
These parameters are deprecated and it is documented in OCS guide.
We only use 
- `openshift_storage_glusterfs_image`
- `openshift_storage_glusterfs_block_image`
- `openshift_storage_glusterfs_s3_image`
- `openshift_storage_glusterfs_heketi_image`
Refer this bz- https://bugzilla.redhat.com/show_bug.cgi?id=1685161 for more info.